### ISSUE TITLE
[Bug]: Invite links via E-Mail not working

### ISSUE THREAD

**Issue by pReya** [coollabsio/coolify#3211]
### Description

I invited a user via E-Mail. They got the mail, and the link is correct (same link I see in the admin area). However the link only leads them to the login screen - not to a signup/set password screen.

I also checked whether this depends on the "Registration allowed" setting, but it doesn't same problem, no matter the "registration allowed" setting.

Maybe this is something proxy related? I switched my proxy to Caddy. Do invite links only work with default (Traefik) proxy?

### Minimal Reproduction (if possible, example repository)

- Invite user via E-Mail
- Click link

### Version

v4.0.0-beta.323

### Cloud?

- [ ] Yes
- [X] No

**Comment by H1D** [comment]
I'm also `v4.0.0-beta.323` and it didn't work until I set up SMTP server

**Comment by pReya** [comment]
> I'm also `v4.0.0-beta.323` and it didn't work until I set up SMTP server

Obviously, E-Mails can't be sent without SMTP server ;) This is not what I was talking about here.

**Comment by peaklabs-dev** [comment]
We made some fixes for invite links in one of the last versions. Please re-test this and if this is still an issue, reopen this issue or create a new one.
